.github/CODEOWNERS

```diff
@@ -351,6 +351,8 @@
 /pkg/util/cgroups/                      @DataDog/container-integrations
 /pkg/util/retry/                        @DataDog/container-integrations
 /pkg/util/intern/                       @DataDog/ebpf-platform
+/pkg/util/crashreport/                  @DataDog/windows-agent
```

Comment:
I think that crash report is created by @DataDog/windows-kernel-integrations (cc @derekwbrown)

Reply:
yes, that's correct